wait/host.go

```diff
@@ -0,0 +1,152 @@
+package wait
+
+import (
+	"context"
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"reflect"
+)
+
+type HostAwaitility struct {
```

Comment:
can we add some comments on the public functions?

Reply:
added 3306bea